plugins/services/src/js/reducers/serviceForm/__tests__/EnvironmentVariables-test.js

```diff
@@ -92,5 +92,12 @@ describe('Environment Variables', function () {
         {type: SET, value: 'value', path: ['env', 0, 'value']}
       ]);
     });
+    it('should keep complex values', function () {
```

Comment:
:notsureif: the documentation comment for `applyHook` and `doAction` is the same :trollface: